.github/workflows/dockerhub.yml

```diff
@@ -44,12 +45,27 @@ jobs:
       uses: docker/build-push-action@v2
       with:
         file: ./Dockerfile.barebone
+        context: .
         push: true
         tags: |
           transferwiseworkspace/pipelinewise-barebone:${{ steps.genTag.outputs.replaced }}
           transferwiseworkspace/pipelinewise-barebone:latest
+          
+    - name: Build and push main image with only default connectors
+      id: docker_build_default
+      uses: docker/build-push-action@v2
+      with:
+        file: ./Dockerfile
+        context: .
+        build_args: |
+          "connectors=default"
+        push: true
+        tags: |
+          transferwiseworkspace/pipelinewise-default:${{ steps.genTag.outputs.replaced }}
```

Comment:
can we perhaps keep the same image name `pipelinewise` but have a different tag, say e.g `{{git_tag}}-default` .. similar to how python base image have `3.8-buster`, `3.8-slim` ..etc

Reply:
fixed